.eslintrc.js

```diff
@@ -15,7 +15,12 @@ module.exports = {
     'import/no-unresolved': 0,
     'no-await-in-loop': 0,
     'new-cap': 0,
-    'max-len': ['error', {'code': 250}],
+    'max-len': [ 'error', { 'code': 250 } ],
     'linebreak-style': 'off',
+    'object-curly-spacing': [ 'error', 'always' ],
+    'space-in-parens': [ 'error', 'always' ],
+    'array-bracket-spacing': [ 'error', 'always' ],
```

Comment:
Remove this rule. No need to add this. Recommended to add this rule`object-curly-spacing`  only.

Reply:
Done